pyproject.toml

```diff
@@ -57,6 +57,8 @@ dependencies = [
     "psutil >= 5.9.5", # may remove later
     "rich == 13.5.2",
     # TODO: Test on cpu-only machine if [cuda] dependency is an issue
+    # FIXME: Required for latest tritonclient 2.46.0
+    "grpcio>=1.64.0",
```

Comment:
Just noticed we need to set the `TRITON_CLI_REPO_TAG` in the github action to trigger the pipeline. I think it can be fixed by adding this around [here](https://github.com/triton-inference-server/triton_cli/blob/3f61b39888c2359a3549964dc447667049e5e5f3/.github/workflows/trigger_ci.yml#L20C88-L20C106):
```
-F variables[TRITON_CLI_REPO_TAG]=${GITHUB_HEAD_REF}
```